content/project/panorama.md

```diff
@@ -1,16 +1,44 @@
 ---
-title: "Panorama"
+title: "PANORAMA"
 date: 2020-03-11T14:40:20-04:00
 logo: "/images/research/panorama.png"
-tags: ["System of Systems", "Modeling"]
+tags: ["System of Systems", "Modeling" "Embedded systems" "Automotive and Aerospace Systems design"]
 homepage: "https://www.panorama-research.org/"
 facebook: ""
 linkedin: ""
 twitter: "https://twitter.com/PanoramaEng"
 youtube: ""
 funding_bodies: ["itea3", "bmbf"]
-eclipse_projects: ["technology.app4mc"]
+eclipse_projects: ["technology.app4mc" "modeling.capra"]
```

Comment:
Sme issue here.

Reply:
Added the following commits to fix this issues. Thx

Reply:
thx